lib/elixir/lib/float.ex

```diff
@@ -198,7 +198,7 @@ defmodule Float do
   defp add_dot(acc, false), do: acc <> ".0"
 
   @doc """
-  Rounds a float to the largest number less than or equal to `num`.
+  Rounds a float to the largest float less than or equal to `number`.
```

Comment:
```suggestion
  Rounds a float to the largest number less than or equal to `number`.
```

Reply:
This returns a float, so calling this "float" is the right way to go IMO.

Reply:
The statement isn't true, though. The biggest float smaller than 13.1 isn't 13.0